orttraining/orttraining/python/ort_trainer.py

```diff
@@ -476,6 +477,7 @@ def create_ort_training_session_bind_parameters(model, device, world_rank=-1, wo
     ort_parameters.world_rank = world_rank
     ort_parameters.world_size = world_size
     ort_parameters.gradient_accumulation_steps = gradient_accumulation_steps
+    ort_parameters.set_gradients_as_graph_outputs = True
```

Comment:
One tiny question. Why is this True here (line 480) and False at line 407? The c++ code seems to assume false as the default

Reply:
While using easy mode, we need the gradients as outputs, so setting it explicitly to True here. 
It is explicitly set to False for non-easy mode (using ORTTrainer) for clarification, even though the C++ default is False already.